### PR TITLE
docs(content/update): Add Weld Issue Workaround Note

### PR DESCRIPTION
**Pull request by psavidis**
Related-to: https://github.com/camunda/camunda-bpm-platform/issues/4434

### CODE REVIEW

File: content/update/patch-level.md [hunk 1]
```diff
@@ -493,12 +493,19 @@ This API is an internal API, which means it's **not** part of the public [REST A
...
 `Tomcat 10` is compatible with the `jakarta` namespace. 
 If you wish to use it, the `jakarta` modules needs to be used (`camunda-webapp-tomcat-jakarta`, `camunda-engine-rest-jakarta`).
 
 The `javax` modules won't work with `Tomcat 10`.
 
 For detailed steps of manual installation, follow the [Tomcat Manual Installation Guide]({{< ref "/installation/full/tomcat/manual.md" >}})
+
+**<h4>Weld Classloading Issues</h4>**
+In deployment scenarios involving one or more process applications with managed beans, classloading issues may occur if the WELD library is directly embedded in the WAR or process application deployment.
+To resolve this, it is recommended to relocate the WELD library from the WAR or process application deployment to the Tomcat server's library folder.
```
Comment (danielkelemen) [inline content/update/patch-level.md:508]: ❓ Do they actually need to relocate this or just not add it in their WARs? Don't we already ship it in our distros?

**Review by danielkelemen — COMMENTED** [review]
👍 Looks pretty good. Added some suggestions and one question.

File: content/update/patch-level.md (hunk 1 above)
Comment (psavidis) [reply content/update/patch-level.md:508]: The message is emphasising on the workaround to move the `weld` from the `WEB-INF/lib` folder to the `$CATALINA_HOME/lib` folder to fix the class loading issues. It doesn't make any assumption if the user will use our installation or a fresh Tomcat server etc.

File: content/update/minor/721-to-722/_index.md [hunk 2]
```diff
@@ -152,4 +152,9 @@ Replace the artifact `camunda-engine-rest-$PLATFORM_VERSION-tomcat.war` with `ca
 
 ### Migrating to the Tomcat 10 Docker Image
 
-If your application uses a Docker image based on `Tomcat 9`, you need to perform the above migration steps yourself before your application is compatible with the `jakarta` namespace changes the new Tomcat version introduces.
\ No newline at end of file
+If your application uses a Docker image based on `Tomcat 9`, you need to perform the above migration steps yourself before your application is compatible with the `jakarta` namespace changes the new Tomcat version introduces.
+
+### CDI and Weld issues
```
[the remark this answers is not in the document]
Comment (psavidis) [reply content/update/minor/721-to-722/_index.md]: Done

**Review by danielkelemen — APPROVED** [review]

**Pull request merged**